cmd/http.go

```diff
@@ -89,14 +89,20 @@ Examples:
   # HTTP HEAD request to jsdelivr.com from 2 probes in New York (protocol, port and path are inferred from the URL)
   http https://www.jsdelivr.com:443/package/npm/test?nav=stats from New York --limit 2
 
-  # HTTP GET request to google.com from 2 probes from London or Belgium
-  http google.com from London,Belgium --limit 2 --method get
+  # HTTP GET request to google.com from 2 probes from London or Belgium in CI mode
+  http google.com from London,Belgium --limit 2 --method get --ci
 
   # HTTP HEAD request to jsdelivr.com from a probe that is from the AWS network and is located in Montreal using HTTP2
   http jsdelivr.com from aws+montreal --protocol http2
 
-  # HTTP GET request google.com with ASN 12345 with json output
-  http google.com from 12345 --json`,
+  # HTTP HEAD request to jsdelivr.com from a probe that is located in Paris, using the /robots.txt path with "test=1" query string
+  http jsdelivr.com from Paris --path /robots.txt --query "test=1"
```

Comment:
You add a path example here but its probably not clear what is the difference. We should mention somewhere that there are 2 methods of building the command. The short with the full URL provided and auto-parsed or long with each parameter manually supplied, probably fitted best for scripting.

Reply:
Added extra info on this